django_celery_monitor/admin.py

```diff
@@ -191,6 +194,25 @@ def kill_tasks(self, request, queryset):
                 revoke(state.task_id, connection=connection,
                        terminate=True, signal='KILL')
 
+    @action(_('Retry selected tasks'))
+    def retry_tasks(self, request, queryset):
+        with current_app.default_connection() as connection:
+            for state in queryset:
+                module_name, task_name = state.name.rsplit('.', 1)
+                task = getattr(import_module(module_name), task_name, None)
+                if task:
+                    kwargs = {
+                        'args': ast.literal_eval(state.args),
+                        'kwargs': ast.literal_eval(state.kwargs),
```

Comment:
`ast.literal_eval` may work for simple arguments, but it's a different thing for more complex objects that can be serialized but may not be able to evaluated by the ast module. I wonder if there is a way for catching ValueError exceptions or whatever a faulty evaluation may cause and showing to the user in the admin that the retry didn't work because it couldn't parse the arguments? Alternatively, would using Celery's serialization system make this less brittle since we could use it to deserialize the argument values?